test/subscription.js

```diff
@@ -1751,3 +1752,33 @@ test('`withFilter` tool works with async filters', t => {
     })
   })
 })
+
+test('subscription server does not register fastify-websocket if already registered', t => {
```

Comment:
could you please expand this test by using the graphql subscription and adding another non-graphql websocket route?

Reply:
ok!

Reply:
@mcollina I've added the test, so could you check it again? Thank you for your review!